pkg/receive/config.go

```diff
@@ -75,6 +77,18 @@ func NewConfigWatcher(logger log.Logger, r prometheus.Registerer, path string, i
 				Name: "thanos_receive_hashrings_file_refreshes_total",
 				Help: "The number of refreshes of the hashrings configuration file.",
 			}),
+		hashringNodesGauge: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "thanos_receive_hashring_nodes",
+				Help: "The number of nodes per hashring.",
+			},
+			[]string{"name"}),
+		hashringTenantsGauge: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "thanos_receive_hashring_tenants",
+				Help: "The number of tenants per hashring.",
+			},
+			[]string{"name"}),
```

Comment:
Just curious on cardinality of this - this change on restart time or is fairly consistent across restarts?

Reply:
This is consistent across restarts. `Name` comes from a manually maintained list which is fed through a configuration file.